addons/mail/wizard/mail_template_preview.py

```diff
@@ -43,7 +43,7 @@ def default_get(self, fields):
                            help="Comma-separated recipient addresses")
     email_cc = fields.Char('Cc', compute='_compute_mail_template_fields', help="Carbon copy recipients")
     reply_to = fields.Char('Reply-To', compute='_compute_mail_template_fields', help="Preferred response address")
-    scheduled_date = fields.Char('Scheduled Date', compute='_compute_mail_template_fields',
+    scheduled_date = fields.Datetime('Scheduled Date', compute='_compute_mail_template_fields',
```

Comment:
technically template field is a char ... I think preview should also be a char as it displays template value if no rendering record is found (see _set_mail_attributes). It could lead to trying to set jinja into a datetime field.